src/main/java/com/google/devtools/build/lib/util/ExitCode.java

```diff
@@ -69,6 +69,9 @@ public final class ExitCode {
       ExitCode.create(45, "PERSISTENT_BUILD_EVENT_SERVICE_UPLOAD_ERROR");
   public static final ExitCode EXTERNAL_DEPS_ERROR = ExitCode.create(48, "EXTERNAL_DEPS_ERROR");
 
+  public static final ExitCode ANALYSIS_CACHE_DISCARD_FAILURE =
```

Comment:
It's not clear if the pattern is that failures are lower numbers, and errors start at 32? Should this be 10?

Reply:
Any chance this might fall under error code 2, or even 1? 

More broadly, do you consider it important that this is its own distinct error code?

Reply:
I considered it important as I'd like to report on this condition specifically without having to parse the textual output of the build events or otherwise.
If we don't feel that use cause is important, then yeah 1 seems to fit (although I'd much prefer a way to detect this case 🙂)